bluepysnap/node_sets.py

```diff
@@ -89,6 +90,28 @@ def from_dict(cls, content):
         """Create NodeSets instance from a dict."""
         return cls.from_string(json.dumps(content))
 
+    @property
+    def content(self):
+        """Access (a copy of) the node sets contents."""
+        return copy.deepcopy(self._content)
+
+    @property
+    def to_libsonata(self):
+        """Libsonata instance of the NodeSets."""
+        return self._instance
+
+    def update(self, node_sets):
+        """Update the contents of the node set."""
```

Comment:
It should be mentioned in the docstring that `update` returns a set with the names of overwritten node_sets.

Reply:
Agreed -> done